index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Mobile First</title>
+  </head>
+
+  <body>
+    <!-- Header-section starts here -->
+ <div class="hero-section">
+
+    <header>
+      <div class="container">
+       
+        <div class="mobile-status">
+          <div class="left-side">
+            <h4>9:14</h4>
+          </div>
+          <div class="right-side">
+              <img src="/images/status-icons.svg" alt="status icons" />
+            </div>
+          </div>
+          <nav class="navbar">
+            <div class="logo">
+              <h3>MOZO</h3>
+            </div>
+            <div class="menu-bar">
+              <img src="/images/menu-bar.png" alt="menu-bar" />
+            </div>
+          </nav>
+        </div>
+      </header>
+
+       <!-- Header-section Ends here -->
+      <!-- Home-page section Starts here -->
+     <main>
+        <section class="home-page container">
```

Comment:
- Please, let's add the missing background image to this section. see the image below:


![Screenshot from 2022-10-26 23-45-36](https://user-images.githubusercontent.com/59397159/198152832-3bbb36b6-b300-49a2-bbd5-b3852c9c57fc.png)

Reply:
![Screenshot 2022-10-27 at 02 42 32](https://user-images.githubusercontent.com/106531695/198161020-9bfa10a5-c759-4ca4-b8ca-300d06d8e4af.png)